tst/CTA.Rules.Test/UtilsTest.cs

```diff
@@ -112,5 +113,12 @@ public void ThreadSafeExportStringToFile_Permits_Concurrent_Processes()
             // Confirm that text was written to file
             Assert.AreEqual(content, File.ReadAllText(filePath));
         }
+
+        [Test]
+        public void Test_Is_VisualBasic_Project()
+        {
+            Assert.IsFalse(VisualBasicUtils.IsVisualBasicProject("test.csproj"));
+            Assert.IsTrue(VisualBasicUtils.IsVisualBasicProject("C://user/john/repos/test.vbproj"));
```

Comment:
Add test case where `vbproj` is in the path or name of the file (not in the extension)

Reply:
done